### PR TITLE
Awesome Books: with ES6

### PR DESCRIPTION
**Pull request by PraisesPJMT**
# Awesome Books: with ES6
## Project Description
Restructure the Awesome books app code. The goal is to make it more organized by using modules.
## Feature Description
- Recreated the Awesome Book App
- Refactor methods and functions to arrow functions.
- Used [Luxon](https://moment.github.io/luxon/#/install?id=es6) to dynamically apply concurrent date and time.
## Features were tested with the following browsers

- [X] Chrome
- [X] Firefox
- [X] Edge
- [ ] Internet Explorer
- [ ] Safari

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// import { list, bookListSection} from '../index.js';
```
Comment (topeogunleye) [inline modules/books.js]: - [x] I kindly suggest that you do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @PraisesPJMT, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your @topeogunleye so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/books.js (hunk 1 above)
Comment (PraisesPJMT) [reply modules/books.js]: Ow Yess! It was a careless oversight! I just removed the comment!

Thanks so much for the recommendation!

**Comment by PraisesPJMT** [comment]
## Implement Review Change
- Removed all unnecessary comments from all files as recommended by code review!

**Review by Kingstalux — APPROVED** [review]
Hi @PraisesPJMT,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966
 

Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**